src/index.ts

```diff
@@ -86,9 +87,9 @@ export async function findManyCursorConnection<
   const endCursor = nodes.length > 0 ? encodeCursor(nodes[nodes.length - 1], options) : undefined
 
   return {
-    edges: nodes.map((node) => ({
-      cursor: encodeCursor(node, options),
-      ...options.nodeToEdge(node),
+    edges: nodes.map((model) => ({
+      cursor: encodeCursor(model, options),
+      ...options.modelToEdge(model),
```

Comment:
If we rename this to `model`, then `nodes` should be called `models` as well.

Reply:
I think it is a good recommendation. I will work on this tonight and update the PR. Thanks